cosmos/core/airflow.py

```diff
@@ -35,6 +35,11 @@ def get_airflow_task(task: Task, dag: DAG, task_group: TaskGroup | None = None)
     for k, v in task.airflow_task_config.items():
         task_kwargs[k] = v
 
+    on_warning_callback = task.extra_context.get("on_warning_callback")
```

Comment:
Do we need to add this here? Why wasn't this necessary when we implemented the Test warning handling?

Reply:
Thanks! this should not require: fixed in https://github.com/astronomer/astronomer-cosmos/pull/1400/commits/8d7ad6637103772753f3e2a61096f2a5d48f0a9a